docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -0,0 +1,146 @@
+# Step 7 - Customize the User forms
+
+We want to have a consistent design over our web site. When using the pagelayout for the user forms, you see the design of your website. We want to welcome our new users with a pretty register page.
+
+## Use custom template for user forms
+
+
+At the same level than `pagelayout` add a `user_registration` directive so the beginning of your `views.yml` file looks like this:
+
+```yaml
+ezpublish:
+    system:
+        default:
+            pagelayout: pagelayout.html.twig
+            user:
+                layout: pagelayout.html.twig
+            user_registration:
+                templates:
+                    form: user/registration_form.html.twig
+                    confirmation: user/registration_confirmation.html.twig
+```
+Then clear the cache with the command we have seen before: `php app/console cache:clear` to see the consequences of the addition you made. 
+## Create the overriding templates files
+
+You will create 3 Twig templates to manage the registration and the login forms, including confirmation page.
+
+ * `registration_form.html.twig`: the main form file
+ * `registration_content_form.html.twig`: the file with the design details for each field
+ * `registration_confirmation.html.twig`: the page the user will see after registration
+
+!!! note
+    All the Twig templates come from the eZPlatform Demo, you can retrieve it on the Github repository: https://github.com/ezsystems/ezplatform-demo/tree/master/app/Resources/views/themes/demo/user
```

Comment:
`you can retrieve them in the Github repository`

or maybe even better: you can find/view them...